crml/erc20-peg/src/lib.rs

```diff
@@ -66,6 +72,16 @@ decl_storage! {
 		AssetIdToErc20 get(fn asset_to_erc20): map hasher(twox_64_concat) AssetId => Option<EthAddress>;
 		/// Metadata for well-known erc20 tokens (symbol, decimals)
 		Erc20Meta get(fn erc20_meta): map hasher(twox_64_concat) EthAddress => Option<(Vec<u8>, u8)>;
+		/// Map from asset_id to minimum amount and delay
+		ClaimDelay get(fn claim_delay): map hasher(twox_64_concat) AssetId => Option<(Balance, T::BlockNumber)>;
+		/// Map from claim id to claim
+		PendingClaims get(fn pending_claims): map hasher(twox_64_concat) ClaimId => Option<PendingClaim>;
+		/// Map from block number to claims scheduled for that block
+		ClaimSchedule get(fn claim_schedule): map hasher(twox_64_concat) T::BlockNumber => Vec<ClaimId>;
```

Comment:
We should use the language 'delay' for this so its clear on intention
```suggestion
		DelayedClaims get(fn pending_claims): map hasher(twox_64_concat) ClaimId => Option<DelayedClaim>;
		/// Map from block number to claims scheduled for that block
		DelayedClaimSchedule get(fn delayed_claim_schedule): map hasher(twox_64_concat) T::BlockNumber => Vec<ClaimId>;
```